.github/workflows/push.yml

```diff
@@ -114,9 +114,10 @@ jobs:
       # Generate tag for operator without "v"
       - name: Generate Operator Tag
         id: generate_operator_tag
-        uses: anothrNick/github-tag-action@1.26.0
+        uses: github-tag-action@1.32.0
```

Comment:
uses: anothrNick/github-tag-action@1.26.0  and new is uses: github-tag-action@1.32.0 is  correct.

Reply:
ouch good catch